packages/contact-center/store/src/store.ts

```diff
@@ -11,20 +19,31 @@ class Store {
     makeAutoObservable(this, {cc: observable.ref});
   }
 
-  init(webexConfig: any, access_token: string): Promise<void> { 
+  registerCC(webex: WithWebex['webex']): Promise<Profile> {
+    this.cc = webex.cc;
+    console.trace(this.cc.register);
+    return this.cc.register().then((response: Profile) => {
+      this.teams = response.teams;
+      this.loginOptions = response.loginVoiceOptions;
+    }).catch((error) => {
+      console.error('Error registering contact center', error);
+    });
+  }
+
+  init(options: InitParams): Promise<void> {
+    if('webex' in options) {
+      return this.registerCC(options.webex);
```

Comment:
So we are assuming that ext devs will call this init only after they have received the 'ready' event. Just emphasizing so that we remember to document this. 

Lets add a comment here so that its clear for anyone who looks at the code.